content/vendor/guides/ha-guide.md

```diff
@@ -0,0 +1,411 @@
+---
+date: "2020-02-20T00:00:00Z"
+lastmod: "2020-02-20T00:00:00Z"
+title: "Deploy an HA Cluster"
+weight: "1005"
+---
+
+In this guide, we'll walk through the steps needed to take enable Kubernetes' high availability capabilities with a cluster installed by kURL and managed by KOTS. 
+This only applies to 'embedded' cluster installs using Kurl.
+
+## Objective
+
+As with the rest of the guides, this is meant as a 'hello world' example to help get you familiarized with the process of standing up a highly available Kubernetes cluster using the Replicated toolset. The architecture we will follow is presented below:
+
+![Architecture](/images/guides/kots/ha-cluster-architecture.png)
+
+The diagram shows 4 Virtual Machines (VMs) running on Google Cloud Platform (GCP), with one of them used solely as a load balancer. Note that there aren't any requirements to use GCP. You can follow along with VMs provisioned on a different cloud provider or on premise.
+
+## Overview
+
+The guide is broken into two parts. The [first part](#part-i---setting-up-the-cluster) will walk you through how to stand up the cluster, while the [second part](#part-ii-testing-scenarions--troubleshooting) will go through a couple of testing scenarios to validate the cluster's resiliency and some troubleshooting tips.
+
+There are a few things to keep in mind about this guide:
+
+- All VMs are going to be created in Google Cloud Platform (GCP) but nothing in this guide is dependent on GCP specific services. 
+- The examples in this guide used ephemeral resources.
+- The Load Balancer we'll use for this exercise is HA Proxy. 
+- The sample application has a database component as described [here](#sample-application), which will help us validate data retention in of our testing scenarios.
+
+## Prerequisites
+
+- This exercise will need 4 VMs that will need to communicate with each other. Please refer to the [Procuring the (virtual) hardware](#provisioning-the-virtual-hardware) section for guidance on system requirements for these. 
+
+- In the example commands and screenshots, we are using a sample application called 'AppDirect'. Please refer to the [Sample Application](#sample-application) section for more details if you wish to follow along and use it.
+
+- This is an advanced topic, so this guide will assume you've already completed one of the [Getting Started Guides](/vendor/guides/#getting-started), and have already a level of familiarity with the Replicated tool set.
+
```

Comment:
This is great, love the organization/presentation so far

Reply:
thanks!